src/load/azext_load/data_plane/utils/argtypes.py

```diff
@@ -341,3 +342,24 @@
         "Example: `--dimension-filters key1=value1 key2=*`, `--dimension-filters *`"
     ),
 )
+
+autostop = CLIArgumentType(
+    validator=validators.validate_autostop_enable_disable,
+    options_list=["--autostop"],
+    type=str,
+    help="Whether auto-stop should be enabled or disabled. The default value is enable.",
+)
+
+autostop_error_rate = CLIArgumentType(
+    options_list=["--autostop-error-rate"],
+    type=Decimal,
+    validator=validators.validate_autostop_error_rate,
+    help="Threshold percentage of errors on which test run should be automatically stopped. Allowed values are in range of 0.0-100.0",
+)
+
+autostop_error_rate_time_window = CLIArgumentType(
+    options_list=["--autostop-time-window"],
+    type=int,
```

Comment:
Can we confirm if we support int only or decimal for the time window ?

Reply:
only integer
![image](https://github.com/user-attachments/assets/e8fd557d-db1c-4054-9e80-84b26ca750fc)